code_schemes/kakuma_household_language.json

```diff
@@ -3,6 +3,62 @@
   "Name": "Kakuma household language",
   "Version": "0.0.0.1",
   "Codes": [
+    {
+      "DisplayText": "english",
+      "VisibleInCoda": true,
+      "NumericValue": 0,
+      "CodeID": "code-d083ae8a",
+      "CodeType": "Normal",
+      "StringValue": "english"
+    },
+    {
+      "DisplayText": "swahili",
+      "VisibleInCoda": true,
+      "NumericValue": 1,
+      "CodeID": "code-17b22de0",
+      "CodeType": "Normal",
+      "StringValue": "swahili"
+    },
+    {
+      "DisplayText": "somali",
+      "VisibleInCoda": true,
+      "NumericValue": 2,
+      "CodeID": "code-0ef426aa",
+      "CodeType": "Normal",
+      "StringValue": "somali"
+    },
+    {
+      "DisplayText": "turkana",
+      "VisibleInCoda": true,
+      "NumericValue": 3,
+      "CodeID": "code-9f7f192b",
+      "CodeType": "Normal",
+      "StringValue": "turkana"
+    },
+    {
+      "DisplayText": "oromo",
+      "VisibleInCoda": true,
+      "NumericValue": 4,
+      "CodeID": "code-58d86c44",
+      "CodeType": "Normal",
+      "StringValue": "oromo"
+    },
+    {
+      "DisplayText": "sudanese",
```

Comment:
(not south sudanese or sudanese/juba arabic etc.?)

Reply:
updated to `sudanese/nuer/juba_arabic`